Makefile

```diff
@@ -118,7 +118,7 @@ ifeq ("$(wildcard $(ENTERPRISE_FE_EXT_FILE))","") ## if enterprise is not enable
 i18n-extract-enterprise:
 	@echo "Skipping i18n extract for Enterprise: not enabled"
 else
-i18n-extract-enterprise: $(SWAGGER) ## Generate API Swagger specification
+i18n-extract-enterprise: ## Generate API Swagger specification
```

Comment:
What does this have to do with Swagger?

Reply:
oh, nothing. Just more copy-pasting i need to clean up 😶‍🌫️